src/app/@theme/styles/styles.scss

```diff
@@ -29,3 +30,26 @@
   }
   @include bootstrap-rtl();
 };
+
+.tree-menu .tree-menu-option:hover {
```

Comment:
let's move these styles to a feature where the tree component is used

Reply:
they didn't work in tree.component.css

Reply:
we definitely shouldn't have same here as global styles. What was the issue?

Reply:
added in local style sheet